packages/core/src/App/app.js

```diff
@@ -25,10 +25,10 @@ const setStorageEvents = root_store => {
     });
 };
 
-const initStore = notification_messages => {
+const initStore = (notification_messages, platform_context) => {
```

Comment:
Adding the comment here as well: possible to pass `is_crypto` as a function argument to where it's necessary in the stores instead of keeping it on the state of the stores?

Reply:
I think it is absolutely possible to do that. But this would require backtracking many of the "computed values" and passing the context from the component. I guess that would create more confusion and a lot more code.

Reply:
True, I guess another option would be to use a store instead of context for the platform state? But i'm not sure if that is better than this.

Reply:
Yes, I thought of that as well. I think we can add another store for that, but that requies components that are not "connected" to the store to be connected, unless we would want to maintain them separately?